test/controllers/teams_controller_test.rb

```diff
@@ -0,0 +1,57 @@
+require "test_helper"
+
+class TeamsControllerTest < ActionController::TestCase
+
+  setup do
+    @team = teams(:one)
+    sign_in users(:nancy)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:teams)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  def test_should_get_settings
+    get :settings, id: @team
+    assert_response :success
+  end
+
+  test "should create team" do
+    assert_difference('Team.count') do
+      post :create, team: { is_personal: @team.is_personal, name: @team.name, notch_count: @team.notch_count, short_name: @team.short_name }
+    end
+
+    assert_redirected_to team_path(assigns(:team))
+  end
+
+  test "should show team" do
+    get :show, id: @team
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @team
+    assert_response :success
+  end
+
+  test "should update team" do
+    patch :update, id: @team, team: { is_personal: @team.is_personal, name: @team.name, notch_count: @team.notch_count, short_name: @team.short_name }
+    assert_redirected_to team_path(assigns(:team))
+  end
+
+  test "should destroy team" do
+    assert_difference('Team.count', -1) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.